daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -238,8 +238,11 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
   })
 
   def qnameConvert(s: String): RefQName = {
-    val eQN = QName.refQNameFromExtendedSyntax(s)
-    eQN.get
+    val eQN = QName.refQNameFromExtendedSyntax(s).recover {
+      case e: Throwable => throw e
```

Comment:
This seems like a no-op. If you don't have the recover, it should propagate this throwable exactly as this does. 

So if you want this, as a place to put a breakpoint, add a comment to that effect. Or take this out once done debugging.